### DISCUSSION TITLE
Link to heading 4

### DISCUSSION THREAD

**Discussion by JasonOui1** [outline/outline#4153] [Ideas]
Is it possible to link to a heading 4 within another document? I have some documents created that have 4 headings and it would be helpful to link back to the heading 4 of those documents. 

I'm posting to ideas as I'm not sure if this would be a feature request or not.

**Comment by Limezy** [comment]
Yes, it's possible, even if a little bit cumbersome.
1. Create your heading with level 1, 2 or 3 (not 4)
2. Show the outline, right clic on the heading and copy link
3. Change your heading to level 4
4. Use the link in another document

**Reply by tommoor** [reply]
Not sure I follow why it's neccessary to create as H3 first?

**Reply by Limezy** [reply]
Because since a recent version, h4 (and h5 and h6) are not shown in the outline anymore ! So there is easy way to know the link

(But I feel it's not a bad thing not to have all the headings showing up in the heading)

**Reply by tommoor** [reply]
You can get the link by clicking on the "#" that appears to the left of the heading when hovering over it

**Reply by JasonOui1** [reply]
I just tested this and you can click the # to copy the link. My apologies that I missed that.

**Reply by Limezy** [reply]
Indeed ! Thanks for the tip
